### ISSUE TITLE
Static member access with variable property name not treated as a use

### ISSUE THREAD

**Issue by Deewiant** [phpmd/phpmd#302]
Example code using a [variable property name](https://secure.php.net/manual/en/language.variables.variable.php) to access a static variable:

``` php
<?php
class Foo {
   public static $member;
   public $member2 = 123;
}
function test($x) {
   return Foo::$member->$x;
}
Foo::$member = new Foo;
var_dump(test('member2'));
?>
```

This code does work:

```
$ php asdf.php
int(123)
```

But results in a false positive:

```
$ phpmd asdf.php text unusedcode

/tmp/asdf.php:6 Avoid unused parameters such as '$x'.
$ phpmd --version
PHPMD 2.2.3
```

**Comment by ravage84** [comment]
Duplicate of #229
